analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -77,7 +77,7 @@ private const val PUB_LOCK_FILE = "pubspec.lock"
 private val flutterCommand = if (Os.isWindows) "flutter.bat" else "flutter"
 private val pubCommand = if (Os.isWindows) "pub.bat" else "pub"
 
-private val flutterVersion = Os.env["FLUTTER_VERSION"] ?: "v1.12.13+hotfix.9-stable"
+private val flutterVersion = Os.env["FLUTTER_VERSION"] ?: "2.0.6-stable"
```

Comment:
Does this work without the next commit? If not, commits should be squashed as each individual commit ideally should pass all tests.